Papr/Commons/Constants.swift

```diff
@@ -117,16 +117,30 @@ enum Constants {
 
             static var squareAndArrowDownMedium: UIImage {
                 if #available(iOS 13.0, *) {
-                    return UIImage(systemName: "square.and.arrow.down", withConfiguration: symbolConfigurationMedium)!
+                    return UIImage(systemName: "arrow.down", withConfiguration: symbolConfigurationMedium)!
                 }
-                return UIImage(imageLiteralResourceName: "down-black")
+                return UIImage(imageLiteralResourceName: "arrow.down")
             }
 
             static var squareAndArrowDownSmall: UIImage {
                 if #available(iOS 13.0, *) {
-                    return UIImage(systemName: "square.and.arrow.down", withConfiguration: symbolConfigurationSmall)!
+                    return UIImage(systemName: "arrow.down", withConfiguration: symbolConfigurationSmall)!
                 }
-                return UIImage(imageLiteralResourceName: "down-black")
+                return UIImage(imageLiteralResourceName: "arrow.down")
+            }
+
+            static var doneWhiteSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "checkmark", withConfiguration: symbolConfigurationSmall)!.withTintColor(.white)
+                }
+                return UIImage(imageLiteralResourceName: "done")
+            }
+
+            static var addSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "plus", withConfiguration: symbolConfigurationMedium)!.withTintColor(.white)
```

Comment:
Can we remove tinting? Otherwise theming wouldn't work on this. 🤔

Reply:
It was previously `addSmallWhite` so I wasn't sure if I should hardcode it or not. Will remove it as well 👍🏻